package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "This project is a simple HTML list of To Do tasks. The list will be styled according to the specifications listed below. This simple web page will be built using webpack and served by a webpack dev server.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open"
+  },
```

Comment:
- [ ] <b>REQUIRED:</b> You are missing one important script that we need in order to be able to build our app. and that is `build`. Kindly add it.